### ISSUE TITLE
API key is displayed as invalid while it's valid in certain scenarios

### ISSUE THREAD

**Issue by Mai-Saad** [wp-media/imagify-plugin#921]
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version => 2.2.3.1
- Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
API key isnot accepted although it is correct if we click anywhere on screen while validating it

**To Reproduce**
Scenario1:
1. fresh install of imagify
2. activate the plugin
3. add key and click connect then click anywhere on screen
4. once done, refresh and hover imagify in admin bar => account data not displayed
5.open imagify settings  => key isnot valid displayed

Scenario2:
1- fresh install and activation
2- use valid key
3- edit the key in settings page to be invalid and save 
4- use the right key and click anywhere outside the field
5- save the right key

**Screenshots**
accept valid key even if we clicked anywhere on screen

**Additional context**
- clear the field and save then put the right key and save without clicking anywhere fixes the problem
**Acceptance Criteria (for WP Media team use only)**
Clear instructions for developers, to be added before the grooming

**Comment by saranshj4** [comment]
https://secure.helpscout.net/conversation/2764122416/523552
https://secure.helpscout.net/conversation/2762809607/523207
https://secure.helpscout.net/conversation/2763037230/523311
https://secure.helpscout.net/conversation/2762982595/523291
https://secure.helpscout.net/conversation/2761637766/522928
https://secure.helpscout.net/conversation/2756448160/521854
https://secure.helpscout.net/conversation/2759598329/522518
https://secure.helpscout.net/conversation/2758349637/522257
https://secure.helpscout.net/conversation/2759287151/522449

**Comment by sandyfzu** [comment]
I cannot reproduce this issue in any of the scenarios. @Mai-Saad Tried as well and couldn't either. Maybe it is already fixed starting from some version?

I tried to reproduce using the 2.3.1-RC1: https://group-onecom.slack.com/archives/C08ECU1CH6Z/p1784701793705239 (Mai tried using latest production one)

@saranshj4 Did you have any recent reports about this?

cc @Miraeld @raahulbasu @MathieuLamiot

**Comment by saranshj4** [comment]
Yes, we get such cases from time to time, but they are easily resolved by reinstallation.
Tickets are tagged by: img_invalid_token

The latest case: https://onecomhelp.zendesk.com/agent/tickets/18124887. I have asked for creds instead of suggesting reinstall.
